pkp/ojs
# pkp/pkp-lib#9456 Add private notes for reviewers

### How to test?

_This test can only be done if the OJS instance has multiple journals on it._

1. Ensure the user on which notes will be used is at least associated to 2 journals.
2. As an editor In journal A: add the reviewer to a submission, then clic on Editorial notes in the reviewer menu. Add both a public and private notes
3. As an editor in journal B: add the same reviewer to a submission. In the add user window, display all information about the reviewer: only the public editorial note add by Editor of journal A is displayed.
4. Clic on Editorial notes in the reviewer menu. Add a private note.
5. As an editor In journal A, add the same reviewer to a new submission, in the add reviewer window, display reviewer details, you should see the private notes of Journal A, not the one from Journal B.
